### ISSUE TITLE
App won't enable

### ISSUE THREAD

**Issue by cjay2021** [nursoda/twofactor_email#226]
The app installed ok, but when I click the button to enable email 2FA it doesn't activate it and the "enable" button just reappears.  Is there a known issue that would cause this to happen or a fix for it?

I've tried removing and reinstalling but the same thing happens.

Thanks

**Comment by nursoda** [comment]
No, unknown and sounds like a general NC issue, not an issue with this app.
I cannot reproduce this, thus closing. I hope that you were able to fix it meanwhile.
Note that if you have ad/js blockers in your browser, first try to switch them off for the setup.

**Comment by nursoda** [comment]
Although this bug was reported first, I keep #298 due to the screenshot (Sorry ;) )
I reconsidered and added/improved exception handling in PR #294 due for review.
